frontend/app/src/components/settings/general/AskUserUponSizeDiscrepancySetting.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+withDefaults(
+  defineProps<{
+    dialog: boolean;
+    confirm: boolean;
```

Comment:
```suggestion
    dialog?: boolean;
    confirm?: boolean;
```
Otherwise the props are required and it gives us a warning

Reply:
updated